data/json/effects.json

```diff
@@ -1910,21 +1912,45 @@
       [ "Your vision improves as your Vitamin A deficiency improves.", "good" ]
     ],
     "max_intensity": 3,
+    "base_mods": { "per_mod": [ -1 ] },
+    "scaling_mods": { "per_mod": [ -2 ] },
     "rating": "bad",
     "blood_analysis_description": "Vitamin A Deficiency"
   },
   {
     "type": "effect_type",
     "id": "hypovitB",
-    "name": [ "B12 deficiency", "Slow healing", "No healing" ],
-    "desc": [ "A lack of vitamin B12 in your diet will affect your ability to heal." ],
-    "apply_message": "Simple wounds are starting to concern you more than usual.",
-    "remove_message": "Your wounds now heal normally.",
+    "name": [ "B12 deficiency", "B12 anemia", "B12 severe anemia", "B12 dementia" ],
+    "desc": [
+      "A lack of vitamin B12 in your diet causes early symptoms of anemia.",
+      "Prolonged lack of vitamin B12 in your diet causes anemia, loss of taste, problems with focus and fading memory.",
```

Comment:
Do I understand correctly that skill rust is no longer part of this PR?

Reply:
it affects int still, so if skill rust is enabled, as far as I'm aware int makes it worse?